Packs/EclecticIQ/Integrations/EclecticIQ/README.md

```diff
@@ -1,1227 +1,601 @@
-<td style="width: 449px;">Observable ID</td>
-</tr>
-<tr>
-<td style="width: 229px;">EclecticIQ.Domain.Maliciousness</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">Maliciousness confidence level</td>
-</tr>
-<tr>
-<td style="width: 229px;">Domain.Name</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">Domain name that was tested</td>
-</tr>
-<tr>
-<td style="width: 229px;">Domain.Malcious.Vendor</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">For malicious domains, the vendor that made the decision</td>
-</tr>
-<tr>
-<td style="width: 229px;">Domain.Malcious.Description</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">For malicious domains, the reason that the vendor made the decision</td>
-</tr>
-<tr>
-<td style="width: 229px;">DBotScore.Type</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">Indicator type</td>
-</tr>
-<tr>
-<td style="width: 229px;">DBotScore.Vendor</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">Vendor used to calculate the score</td>
-</tr>
-<tr>
-<td style="width: 229px;">DBotScore.Score</td>
-<td style="width: 62px;">Number</td>
-<td style="width: 449px;">The actual score</td>
-</tr>
-<tr>
-<td style="width: 229px;">DBotScore.Indicator</td>
-<td style="width: 62px;">String</td>
-<td style="width: 449px;">The indicator that was tested</td>
-</tr>
-</tbody>
-</table>
-</div>
-</div>
-<p> </p>
-<div class="cl-preview-section">
-<h5 id="command-example-5">Command Example</h5>
-</div>
-<div class="cl-preview-section">
-<p><code>domain domain=gooc.om</code></p>
-</div>
-<div class="cl-preview-section">
-<h5 id="context-example-5">Context Example</h5>
-</div>
-<div class="cl-preview-section">
-<pre>{
-    "Domain": [
-        {
-            "Name": "gooc.om"
-        }
-    ], 
-    "DBotScore": {
-        "Vendor": "EclecticIQ", 
-        "Indicator": "gooc.om", 
-        "Score": 0, 
-        "Type": "domain"
-    }, 
-    "EclecticIQ.Domain": [
-        {
-            "Maliciousness": "unknown", 
-            "Name": "gooc.om", 
-            "ID": 74, 
-            "LastUpdated": "2018-11-21T13:34:38.964435+00:00", 
-            "Created": "2018-11-21T13:34:38.964435+00:00"
-        }
-    ]
-}
-</pre>
-</div>
-<div class="cl-preview-section">
-<h5 id="human-readable-output-5">Human Readable Output</h5>
-</div>
-<div class="cl-preview-section">
-<h3 id="eclecticiq-domain-reputation---gooc.om">EclecticIQ Domain reputation - gooc.om
-</h3>
-</div>
-<div class="cl-preview-section">
-<div class="table-wrapper">
-<table>
-<thead>
-<tr>
-<th>ID</th>
-<th>Maliciousness</th>
-<th>Name</th>
-<th>LastUpdated</th>
-<th>Created</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>74</td>
-<td>unknown</td>
-<td>gooc.om</td>
-<td>2018-11-21T13:34:38.964435+00:00</td>
-<td>2018-11-21T13:34:38.964435+00:00</td>
-</tr>
-</tbody>
-</table>
-</div>
-</div>
\ No newline at end of file
+# EclecticIQ Cortex XSOAR User Manual
```

Comment:
for adopting the pack, add the below line to the top of the readme file:
Note: Support for this Pack will be moved to the Partner on MONTH, DAY, YEAR.

change the date to the one agreed

Reply:
Updated